validator/validator-main.protoascii

```diff
@@ -1235,6 +1235,14 @@ tags: {
   attrs: { name: "align" }
   attr_lists: "cite-attr"
 }
+tags: {
+  html_format: AMP
+  tag_name: "BLOCKQUOTE"
```

Comment:
I would lean towards putting this in the extension-specific protoascii (to keep extension-specific details out of main, so it doesn't explode in size, and to keep related specs together).

Some precedent below, but I could be swayed if you think this is different.

https://github.com/ampproject/amphtml/blob/cd50214b9afa811aee98b4a7640bdceed0b6e1f9/extensions/amp-nested-menu/validator-amp-nested-menu.protoascii#L80-L86

https://github.com/ampproject/amphtml/blob/cd50214b9afa811aee98b4a7640bdceed0b6e1f9/extensions/amp-script/validator-amp-script.protoascii#L237-L245

https://github.com/ampproject/amphtml/blob/0320666d03db37c4e0e363b55f828aa807ae321c/extensions/amp-list/validator-amp-list.protoascii#L118-L130

Reply:
Done! Thanks.